outputs.tf

```diff
@@ -18,11 +18,20 @@ output "vision_url" {
   description = "IBM Visual Insights URL"
 }
 
+output "vision_username" {
+  value = "admin"
+  description = "IBM Visual Insights username to login"
```

Comment:
"username to login" is redundant. :-) Please change to "IBM Visual Insights username"